### DISCUSSION TITLE
Make graphics drivers better

### DISCUSSION THREAD

**Discussion by raphm4044** [CosmosOS/Cosmos#2413] [Ideas]
Hello

The Cosmos graphics drivers (VBE, VGA and SVGA) are very meh.

VBE : 2 FPS
VGA : Only 3 screen modes
SVGA : FPS are very meh when drawing too much pixels

Please make the Cosmos graphics drivers better

**Comment by selkij** [comment]
the VGA is fine

**Reply by AAM1075** [reply]
Lmao it runs at like 0.2 fps
Very slow

**Reply by zarlo** [reply]
and you using the driver or canvas

**Reply by raphm4044** [reply]
svga driver

**Comment by zarlo** [comment]
most of the issues with them is the canvas system and how people are using them (ie re rendering the hole screen)

**Reply by raphm4044** [reply]
i have created my own graphics driver with svga, and the fps is meh

**Comment by pleasenoban** [comment]
svga is pretty good imo
